python/whylogs/viz/utils/drift_calculations.py

```diff
@@ -1,16 +1,24 @@
-from typing import Dict, Optional
+import warnings
+from typing import Dict, Optional, Union
 
 import numpy as np
 from scipy import stats  # type: ignore
+from scipy.spatial.distance import euclidean
 from typing_extensions import TypedDict
 from whylogs_sketching import kll_doubles_sketch  # type: ignore
 
+from whylogs.core.view.column_profile_view import ColumnProfileView  # type: ignore
 from whylogs.core.view.dataset_profile_view import DatasetProfileView  # type: ignore
 from whylogs.viz.utils.frequent_items_calculations import (
     FrequentStats,
     get_frequent_stats,
     zero_padding_frequent_items,
 )
+from whylogs.viz.utils.histogram_calculations import (
+    HIST_AVG_NUMBER_PER_BUCKET,
+    MAX_HIST_BUCKETS,
+    _calculate_bins,
```

Comment:
won't this come from `whylogs.viz.utils` instead of histogram_calculations?

Reply:
Consider movingh those constants and `_calculate_bins` to the drift_calculations.py file (or some common location). One concern is that we may require the [viz] extra in order to use this method programmatically.

Reply:
Not sure this is what you meant, but yes, to use hellinger calculation programmatically, we'll currently need to install and import [viz]. Everything drift-related ended up in the viz module. We should discuss if this is the correct approach.

Reply:
done in the latest commits. we can open another one if something needs to be done around this comment:

`One concern is that we may require other extras in order to use this method programmatically?`